src/commands/limits/recordcounts/display.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { Messages, SfError } from '@salesforce/core';
+import { Flags, loglevel, orgApiVersionFlagWithDeprecations, SfCommand } from '@salesforce/sf-plugins-core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-limits', 'recordcounts');
+
+export type RecordCount = {
+  name: string;
+  count: number;
+};
+
+export type RecordCounts = RecordCount[];
+
+interface Result {
+  sObjects: RecordCounts;
+}
+
+export class LimitsRecordCountsDisplayCommand extends SfCommand<RecordCounts> {
+  public static readonly aliases = ['force:limits:recordcounts:display', 'org:list:sobject:record-counts'];
+  public static readonly summary = messages.getMessage('description');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+
+  public static flags = {
+    sobject: Flags.string({
+      char: 's',
+      summary: messages.getMessage('sobjectFlagDescription'),
+      aliases: ['sobjecttype'],
+      multiple: true,
+      default: [],
+    }),
+    'target-org': Flags.requiredOrg({
+      char: 'o',
+      aliases: ['targetusername', 'u'],
+      summary: messages.getMessage('targetOrg'),
+    }),
+    'api-version': orgApiVersionFlagWithDeprecations,
+    loglevel,
+  };
+
+  public async run(): Promise<RecordCounts> {
+    try {
+      const { flags } = await this.parse(LimitsRecordCountsDisplayCommand);
+      const conn = flags['target-org'].getConnection(flags['api-version']);
+      const sobjectsPassed = flags.sobject.map((sobject) => sobject.split(',')).flat();
```

Comment:
this is a nice way to handle the transition from stringArray flags to `multiple`.